src/Rinds/FailRuleProcessor.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Rule processor that fails.
+ *
+ * @package WooCommerce Admin/Classes;
+ */
+
+namespace Automattic\WooCommerce\Admin\Rinds;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Rule processor that fails.
+ */
+class FailRuleProcessor {
```

Comment:
The idea of this was to give the developers of the JSON file (WooCommerce.com) an easy way to void a spec and action the note, since just deleting the spec orphans the note.